python/ray/rllib/optimizers/multi_gpu_optimizer.py

```diff
@@ -65,8 +65,7 @@ def _init(self, sgd_batch_size=128, sgd_stepsize=5e-5, num_sgd_iter=10,
         # all of the device copies are created.
         with self.local_evaluator.tf_sess.graph.as_default():
             with self.local_evaluator.tf_sess.as_default():
-                main_scope = tf.get_variable_scope()
-                with tf.variable_scope(main_scope, reuse=tf.AUTO_REUSE):
+                with tf.variable_scope("default", reuse=tf.AUTO_REUSE):
```

Comment:
Note that for multiagent we'll have to replace "default" with the individual policy scopes.